db/init.sql

```diff
@@ -13,6 +13,10 @@ CREATE TABLE IF NOT EXISTS users (
   privacy varchar(255),
   agree_toc varchar(255),
   referral VARCHAR(32),
+  kitns_daily INT DEFAULT 0,
```

Comment:
Are you sure we need so many fields? We did not discussed the design for that.

Reply:
We discussed it earlier with Robin. If you take a look at the KITN screen you'll see four numbers.
![Simulator Screenshot - iPhone 15 Pro - 2023-12-22 at 16 05 22](https://github.com/cleanappio/cleanapp_back_end_v2/assets/20050859/bbfb757a-afd0-4fa0-803d-c131c6611bf8). We might also change that in the future.
So we have to store a detailed info about each kind of KITN count.

Reply:
Ok,  think this design is too heavy, but ok for now.